Source/CesiumRuntime/Public/CesiumMetadataValue.h

```diff
@@ -859,3 +860,39 @@ class CESIUMRUNTIME_API UCesiumMetadataValueBlueprintLibrary
   static TMap<FString, FString>
   GetValuesAsStrings(const TMap<FString, FCesiumMetadataValue>& Values);
 };
+
+/// Grants access to metadata value types not currently supported in blueprint.
+/// Can be useful in C++ code.
+/// Should be moved to UCesiumMetadataValueBlueprintLibrary if those types
+/// become compatible with blueprints in the future.
+class CESIUMRUNTIME_API FCesiumMetadataValueAccess {
```

Comment:
We can drop the letter prefix from the name since this isn't a `USTRUCT` (F) or `UCLASS` (U)

```suggestion
class CESIUMRUNTIME_API CesiumMetadataValueAccess {
```